### PR TITLE
Feature/update model

### PR DESCRIPTION
**Pull request by dirtyg2120**
Update `TwitterUser`, `Tweet`, `Report` due to #34 

New thing which is different from `Tweet` table in #34 is: `tweepy.ReferencedTweet`. It has `type` and `id`, and `type` includes: reply, quote, retweet.

I think ML model and response should be done in PR for ML, so it's not here.

### CODE REVIEW

**Review by produdez — APPROVED** [review]

File: app/models/twitter.py [hunk 1]
```diff
@@ -19,6 +21,7 @@ class Tweet(EmbeddedDocument):
     tweet_id = StringField(primary_key=True)
     text: str = StringField(required=True)
     created_at: datetime = DateTimeField(required=True)
+    referenced_tweets: tweepy.ReferencedTweet = ListField()
```
Comment (phhai-hcmut) [inline app/models/twitter.py:24]: Is this a single tweet or a list of referenced tweets?

Comment (dirtyg2120) [reply app/models/twitter.py:24]: @phhai-hcmut in the docs, it said "A list of Tweets this Tweet refers to"

Comment (phhai-hcmut) [reply app/models/twitter.py:24]: But the type of `referenced_tweets` you specified here is just `tweepy.ReferencedTweet`

Comment (dirtyg2120) [reply app/models/twitter.py:24]: the type of `tweepy.ReferencedTweet` return a list itself, i think. I tried to wrap it inside a `List[]` but it got wrong

Comment (phhai-hcmut) [reply app/models/twitter.py:24]: [This](https://github.com/tweepy/tweepy/blob/v4.4.0/tweepy/tweet.py#L76) say otherwise

Comment (produdez) [reply app/models/twitter.py:24]: fight fight fight 🍾

Comment (dirtyg2120) [reply app/models/twitter.py:24]: i got this when print itself and type
```shell
<class 'list'>
[<ReferencedTweet id=1507148569513537540 type=quoted]
```

Comment (phhai-hcmut) [reply app/models/twitter.py:24]: I means the correct type for `referenced_tweets` is `List[tweepy.ReferencedTweet]`

Comment (dirtyg2120) [reply app/models/twitter.py:24]: let me try again :V

**Comment by phhai-hcmut** [comment]
@dirtyg2120 Higher score mean more bot like or more human like?

File: app/models/twitter.py (hunk 1 above)
Comment (dirtyg2120) [reply app/models/twitter.py]: fixed

**Comment by dirtyg2120** [comment]
> @dirtyg2120 Higher score mean more bot like or more human like?

dunno, Đôrê didn't give me anything :))) except these models

**Review by phhai-hcmut — APPROVED** [review]
